platform/logging/appenders/file/FileAppender.ts

```diff
@@ -0,0 +1,42 @@
+import { createWriteStream, WriteStream } from 'fs';
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { FileAppenderConfig } from './FileAppenderConfig';
+
+export class FileAppender extends BaseAppender {
```

Comment:
Probably neither of file-based appenders will go into the first PR to make it simpler for review, but I'm keeping them right now just to have a broader picture.